inputfiles/overridingTypes.json

```diff
@@ -1,5 +1,40 @@
 [
     {
+        "kind": "typedinterface",
```

Comment:
i do not think you need a new kind here.. just add a property `typeParameters`

Reply:
So would `kind` be `interface` and then `parameters` would become `typeParameters`?